modules/util/portal-tools-rest-builder/src/main/resources/com/liferay/portal/tools/rest/builder/dependencies/dto.ftl

```diff
@@ -281,6 +290,27 @@ public class ${schemaName} <#if dtoParentClassName?has_content>extends ${dtoPare
 		return Objects.equals(toString(), ${schemaVarName}.toString());
 	}
 
+	<#if generateGetValues?has_content>
```

Comment:
I think we need to update this condition, as we don't want to generate for all the entities, only for the ones that has a property with `x-json-map` set